### PR TITLE
Desktop-version

### PR DESCRIPTION
**Pull request by henrykgggg**
## Description
In this PR, I created a Desktop version 

### General requirements

- [x] There are no linter errors.
- [x] Followed the **Github flow**.
- [x] The documentation is professional.

### Milestone requirements, We:

- [x] used queries to create a desktop version
- [x]  used CSS grid and flexbox
- [x]  used images and background from the figma

 # Happy reviewing 😃

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,634 @@
...
+  justify-content: center;
+  align-items: center;
+  gap: 30px;
+  width: 100%;
+  padding: 100px 0;
+  background-color: #f7f7f9;
+}
+
+.card-contain {
+  border: 1px solid #7a869a;
```
Comment (prolajumokeoni) [inline style.css:162]: - [ ] Kindly add the right border color `#DFE1E6` for each card 👍🏿

File: style.css [hunk 2]
```diff
@@ -0,0 +1,634 @@
...
+    width: 100%;
+    height: auto;
+  }
+
+  .about-me-holder section {
+    flex-flow: row;
+  }
+
+  .about-me-holder section:nth-child(1) {
+    width: 45%;
```
Comment (prolajumokeoni) [inline style.css:570]: -  Good job achieving this 👍🏿 
- [ ] The first block which houses the  about myself div should be 40%

File: style.css [hunk 3]
```diff
@@ -0,0 +1,634 @@
...
+  }
+
+  .about-me-holder section:nth-child(2) {
+    width: 50%;
```
Comment (prolajumokeoni) [inline style.css:574]: - [ ] The second block which houses the Languages and skills should be 60%, please update these changes so your design looks similar to the Figma design 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @henrykgggg 👋🏿 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ GitHub flow was correctly used
✔️  Good design



## Required Changes ♻️
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,348 @@
...
+                  <li class="language-item"><span>Css</span></li>
+                </ul>
+              </div>
+              <button type="button" class="click-btn">See project</button>
+            </section>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="about-me">
```
Comment (prolajumokeoni) [inline index.html:215]: - [ ] Kindly add the top border right to your design 👍🏿

File: style.css (hunk 1 above)
Comment (sja-thedude) [reply style.css:163]: - [ ] Optional: Kindly make sure to display the cards correctly as you can see all the cards in even numbers like 2 and 4 follow row reverse as the content is first and then the image. As the previous reviewer forgot to mention this. I have kept it optional but I recommend you implement it in your free time.

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @henrykgggg, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

## Optional - Kindly check the inline comment for it.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**